jbpm-human-task/jbpm-human-task-workitems/src/main/java/org/jbpm/services/task/wih/ExternalTaskEventListener.java

```diff
@@ -46,20 +50,19 @@ public void processTaskState(Task task) {
         RuntimeManager manager = getManager(task);
         RuntimeEngine runtime = manager.getRuntimeEngine(ProcessInstanceIdContext.get(processInstanceId));
         KieSession session = runtime.getKieSession();
-        
+
         if (task.getTaskData().getStatus() == Status.Completed) {
             String userId = task.getTaskData().getActualOwner().getId();
             Map<String, Object> results = new HashMap<String, Object>();
-            
+
             Map<String, Object> taskOutcome = task.getTaskData().getTaskOutputVariables();
             if (taskOutcome != null) {
                 results.putAll(taskOutcome);
-//                results.put("Result", taskOutcome);
             }
 
-        	results.put("ActorId", userId);
```

Comment:
Why was this removev? Is it already included in the `taskOutcome`?

Reply:
This was removed in mistake, reverted.